CFC_DataCollector/recommender/recommendation_pipeline.py

```diff
@@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
         return []
 
     def get_selected_user_utility_model(self, user_id):
-        return []
+        return UserUtilityModel.find_from_db(user_id)
 
     def recommend_trips(self, trip_id, utility_model):
         return []
```

Comment:
I ask because I'm rewriting all of our modelling to use numpy, and so this
is a good time to fix things.

Also, Shankari, do you think we could just wait until Thursday to deploy
onto production? I know that pushing back is not good with such little time
left, but there are going to be some tricky things with deploying. The
current code is fairly inefficient, and so we will need to start collecting
alternatives in batches. In addition, since Josh's code was deleted, the
pipeline will only be generating google maps alternatives right now (he
didn't push a working version last night) and then it will be hard to
convert our db to otp.

I think it would be easiest to do a full end-to-end demo of recommendations
from the baby server on Thursday, and then work on moving to production.

Again, I'm sorry to delay, but since bits and pieces are not ready to go
(notably otp and trip clustering/canonical trips), I think we should wait
on deploying into production.

On Wednesday, April 29, 2015, SHAUN DARSHAN SINGH <sdsingh@berkeley.edu>
wrote:

> Thanks to a silly hack from demo time, recommendations are being generated
> in utility model pipeline.. I'm moving the code over to recommendation
> pipeline however.
>
> Do you remember our discussion a while ago on implementing the user model?
> The issue was that alternatives were not attached to their original trips
> in any way, and so the model could learn crazy things. What was the final
> solution?
>
>
> On Wednesday, April 29, 2015, shankari <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> In CFC_DataCollector/recommender/recommendation_pipeline.py
>> <https://github.com/e-mission/e-mission-server/pull/111#discussion_r29336257>
>> :
>>
>> > @@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
>> >          return []
>> >
>> >      def get_selected_user_utility_model(self, user_id):
>> > -        return []
>> > +        return UserUtilityModel.find_from_db(user_id)
>> >
>> >      def recommend_trips(self, trip_id, utility_model):
>> >          return []
>>
>> @sdsingh <https://github.com/sdsingh>, where are recommendations being
>> generated?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/e-mission/e-mission-server/pull/111/files#r29336257>.
>>
>